### ISSUE TITLE
BUG: Open two draw at the same time will lose function

### ISSUE THREAD

**Issue by windily-cloud** [zsviczian/obsidian-excalidraw-plugin#891]
# Environment

SYSTEM INFO:
	Obsidian version: v1.0.3
	Installer version: v1.0.0
	Operating system: Windows 10 Home China 10.0.22621
	Login status: logged in
	Catalyst license: none
	Insider build toggle: off
	Live preview: on
	Legacy editor: off
	Base theme: dark
	Community theme: Obsidianite
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 1
	Plugins enabled: 1
		1: Excalidraw v1.7.29

# Describe the bug

When I open two draw at the same time in the two leaf, I can't change the theme.

# To Reproduce

1. Create a excalidraw
2. Deconstruct a part of draw
3. Open two draw in the two leaf at the same time
4. I can't change theme

# Expected behavior

When clicking on a menu only the corresponding leaf is activated.

# Screenshots

https://user-images.githubusercontent.com/30640638/202678427-cff28a47-f2db-4479-8f1b-33563a283737.mp4

# Additional context

I followed a recent youtube video tutorial and used a deconstruct script

**Comment by zsviczian** [comment]
I am unable to reproduce the error. What Excalidraw Theme settings are you using?
![image](https://user-images.githubusercontent.com/14358394/202759456-e5b39cd7-9ca0-43ec-adcc-fda47eb90636.png)

**Comment by windily-cloud** [comment]
> I am unable to reproduce the error. What Excalidraw Theme settings are you using?

All settings are default.

I repeatedly test this in the sandbox vault.  Only the following steps were done:

1. Open sandbox vault
2. Install excalidraw
3. Click excalidraw icon in the ribbon menu to create a excalidraw canvas
4. Click excalidraw icon again in the ribbon menu
5. Split Right the excalidraw canvas


https://user-images.githubusercontent.com/30640638/202831504-b287d036-b250-4feb-be96-c5a63c1eab27.mp4

 I made sure I updated obsidian to the latest version, installed the version as well, and also turned off any css snippets and turned off plugins other than excalidraw. All settings are default.

Thx for your reply！！！
